bob-backend/src/pearl/group.rs

```diff
@@ -63,39 +64,43 @@ impl Group {
         }
     }
 
-    pub async fn run(&self, pp: impl Hooks) -> AnyResult<()> {
+    async fn run_under_reinit_lock(&self, pp: impl Hooks) -> AnyResult<()> {
         debug!("{}: read holders from disk", self);
         let config = self.settings.config();
-        let holders = config
+        let new_holders = config
             .try_multiple_times_async(
                 || self.read_vdisk_directory(),
                 "can't create pearl holders",
                 self.settings.config().fail_retry_timeout(),
             )
             .await
             .with_context(|| "backend pearl group read vdisk directory failed")?;
-        debug!("{}: count holders: {}", self, holders.len());
-        if !holders
+        debug!("{}: count holders: {}", self, new_holders.len());
+        if !new_holders
             .iter()
             .any(|holder| holder.gets_into_interval(get_current_timestamp()))
         {
             self.create_current_pearl();
```

Comment:
Can you also fix https://github.com/qoollo/bob/issues/701 as a part of this PR?

Reply:
Actually, I think it is better to delete this call at all. Actual holder will be created on the first put request